src/store/DaoID.ts

```diff
@@ -6,6 +6,7 @@ export default defineStore("DaoStore", {
 		return {
 			dao_id: undefined,
 			govt_id: undefined,
+			proposal_addr: undefined,
```

Comment:
I think we can remove the key initializer of the values which are undefined, since through type declarations. they already are undefined until provided with a value. What do you think.?

Reply:
No, we can't do that. If it contains a value then it won't be undefined until we explicitly set the value to undefined.